pipelines/batch/src/main/java/org/openmrs/analytics/EtlUtils.java

```diff
@@ -35,4 +38,27 @@ static void logMetrics(MetricResults metricResults) {
 		}
 	}
 	
+	static void logMultipleMetrics(List<MetricResults> metricResultsList) {
```

Comment:
I am guessing we won't need this function once we fix the multiple pipeline instance issue.